setup.cfg

```diff
@@ -65,16 +68,16 @@ exclude =
 [options.extras_require]
 testing =
   # upstream
-  pytest >= 7
+  pytest >= 7.4
   pytest-checkdocs >= 2.4
   pytest-flake8 >= 1.0.7
-  coverage >= 6.3
+  coverage >= 7
```

Comment:
LGTM! We are now dropping support for pre-3.9 (as part of #134) Makes sense to bump versions too.